src/css/main.css

```diff
@@ -1,4 +1,85 @@
+
+header {
+    background: #f0f0f0;
+    position: absolute;
+    top: 0;
+    width: 100%;
+}
+
+#header img{
+    position:  absolute;
+    right: 5%;
+    background-color: #f0f0f0;
+    padding: 16px, 16px, 0px, 16px;
+
+}
+
+#header h1 {
+    display: inline;
+    position: left;
+    left: 5%;
+    color: black;
+    position: left;
+    margin: 0%;
+    font-size: 24px;
+    padding: 16px, 16px, 0px, 16px;
+    border: #C0C0C0;
+}
+
+article {
+    padding-top: 150px;
+}
+
 p {
-    color: red;
-    appearance: none;
+    color: black;
+    -webkit-appearance: none;
+       -moz-appearance: none;
+            appearance: none;
+}
+
+ul {
+    list-style-type: none;
+    margin: 0;
+    padding: 0;
+    overflow: hidden;
+    background-color: #C0C0C0;
+    width: 100%;
+}
+
+li {
+    float: left;
+}
+
+li a {
+    display: block;
+    color: black;
+    text-align: center;
+    padding: 16px;
+    text-decoration: none;
+}
+
+h2 {
+    text-align: left;
+    padding-left: 88px;
+}
+
+p.text {
+    padding-left: 88px;
+    padding-right: 88px;
+}
+
+footer {
+    background: #C0C0C0;
+    position: absolute;
+    bottom: 0;
+    width: 100%;
+    padding: 16px;
+}
+
+#footer p.left{
+    float: left;
+}
+
+#footer p.right {
```

Comment:
is there a need for such specificity?

Reply:
I'm not sure if I need the #footer id being called, the class for right is needed for positioning.